engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/ProcessCompiler.scala

```diff
@@ -113,11 +113,11 @@ protected trait ProcessCompilerBase {
 
   private lazy val globalVariablesPreparer = GlobalVariablesPreparer(expressionConfig)
 
-  private def contextWithOnlyGlobalVariables(implicit metaData: MetaData): ValidationContext = {
-    val globalTypes = globalVariablesPreparer.prepareGlobalVariables(metaData).mapValuesNow(_.typ)
+  private def contextWithOnlyGlobalVariables(implicit metaData: MetaData): ValidationContext
+    = globalVariablesPreparer.emptyValidationContext(metaData)
```

Comment:
Can equal be on the of line like it was before or it is some kind of new IDEA standard?

Reply:
ok, I ran formatter for this class...

Reply:
It is not exactly what I meant to. I was thinking about:
```
  private def contextWithOnlyGlobalVariables(implicit metaData: MetaData): ValidationContext =
    globalVariablesPreparer.emptyValidationContext(metaData)
```
It is very similar to the way how you declare in version with `{ }`.